packagedb/api.py

```diff
@@ -517,6 +550,19 @@ class PackageSetViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = PackageSetAPISerializer
 
 
+class PackageWatchViewSet(CreateListRetrieveUpdateViewSet):
+    queryset = PackageWatch.objects.get_queryset().order_by('-id')
+    serializer_class = PackageWatchAPISerializer
+    lookup_field = 'uuid'
```

Comment:
would not the lookup be by PURL?

Reply:
We can use PURL for lookup, should we get rid of uuid altogether?

Reply:
Since we have the both this is your call.

Reply:
Ack, got rid of UUID.